codeforboston/maple
# Hearings scheduled bootstrap refactor

# Summary

closes #1465

refactors css and inline styles into preferable bootstrap when possible and secondarily styled components when necessary.

# Checklist

- [N/A] On the frontend, I've made my strings translate-able.

# Screenshots

![hellyea](https://github.com/codeforboston/maple/assets/73559781/adde98b2-4395-4e71-9b07-5893005db3a3)

# Known issues

N/A

# Steps to test/reproduce

_For each feature or bug fix, create a step by step list for how a reviewer can test it out. E.g.:_

1. goto the main page
2. confirm that it looks dope or nope
